src/datapoint.jl

```diff
@@ -34,14 +34,19 @@ Returns the systematic uncertainty of the data point as a tuple of upper and
 lower bounds. If none is provided it will default to zero.
 """
 syserr(x::DataPoint) = x.sys_err
+
+relstaterr(x::DataPoint) = x.stat_err / abs(x.val)
+
+statbounds(x::DataPoint) = x.val .+ (-1,1).*x.stat_err
```

Comment:
We could create the tuple in the end instead of using broadcasting.
`statbounds(x::DataPoint) = (x.val - x.stat_err,x.val + x.stat_err)`

Reply:
In general `x.stat_err` is a tuple and therefore using the broadcasting. Otherwise your suggestion would need to look like 
` statbounds(x::DataPoint) = (x.val - x.stat_err[1],x.val + x.stat_err[2])` and I don't think that this is a good idea.

Reply:
I see. Then I'm fine with having it this way. We could use `@.` but that is up to your preference.